tests/test_commands.sh

```diff
@@ -23,17 +26,17 @@ rm -r titanic.zip tost sample_submission.csv
 
 echo "kaggle kernels list"
 kaggle k list -m -s Exercise --page-size 5 -p 2 -v  --sort-by dateRun
-kaggle k list --parent stevemessick/exercise-lists
+kaggle k list --parent $KAGGLE_DEVELOPER/exercise-lists
 kaggle k list --competition house-prices-advanced-regression-techniques --page-size 5
 kaggle k list --dataset dansbecker/home-data-for-ml-course --page-size 5
-kaggle k list --user stevemessick --language python --kernel-type notebook --output-type data
+kaggle k list --user $KAGGLE_DEVELOPER --language python --kernel-type notebook --output-type data
 echo "kaggle kernels files"
 kaggle kernels files kerneler/sqlite-global-default -v --page-size=1
 echo "kaggle kernels init"
 kaggle k init -p tests/kernel
 echo "kaggle kernels pull"
-kaggle k pull -p tests/kernel stevemessick/exercise-as-with -m
-kaggle k pull --wp stevemessick/exercise-as-with
+kaggle k pull -p tests/kernel $KAGGLE_DEVELOPER/exercise-as-with -m
+kaggle k pull --wp $KAGGLE_DEVELOPER/exercise-as-with
```

Comment:
These are going to fail unless the `KAGGLE_DEVELOPER` has these notebooks, right?  Should there be a message?  Or use `kaggle k list` to see what's available?  Or just do public notebooks (although maybe the intention is to test the user can access private stuff)?

Reply:
Ideally, it should create the notebook before pulling. As I mentioned in another comment, I'm not sure it is worth the trouble to fix this script.